bin/pycbc_compress_bank

```diff
@@ -57,7 +57,13 @@ parser.add_argument("--low-frequency-cutoff", type=float, default=None,
 pycbc.waveform.bank.add_approximant_arg(parser)
 parser.add_argument("--sample-rate", type=int, required=True,
                     help="Half this value sets the maximum frequency of "
-                    "the compressed waveforms.")
+                    "the compressed waveforms. Must be a power of 2.")
+parser.add_argument("--segment-length", type=int, required=True,
+                    help="The segment length to use for generating the "
+                         "templates for compression, and for calculating "
```

Comment:
The indentation might need to be fixed here. As per PEP8 style guide , continuing lines must be aligned with the opening bracket.